openshift/inspectJobWithCPU-template.yaml

```diff
@@ -0,0 +1,132 @@
+    template: amun-inspect-job-with-cpu
+    app: amun
+    component: amun-inspect
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${AMUN_INSPECTION_ID}"
+      labels:
+        app: amun
+        component: amun-inspect
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: amun
+            component: amun-inspect
+        spec:
+          automountServiceAccountToken: false
+          restartPolicy: Never
+          containers:
+            - name: inspect
+              env: []
+              image: '${AMUN_INSPECTION_ID}:latest'
+              livenessProbe:
+                failureThreshold: 1
+                initialDelaySeconds: 3600
+                periodSeconds: 10
+                tcpSocket:
+                  port: 80
+              resources:
+                limits:
+                  cpu: ${AMUN_CPU}
+                  memory: ${AMUN_MEMORY}
+                requests:
+                  cpu: ${AMUN_CPU}
+                  memory: ${AMUN_MEMORY}
+              volumeMounts:
+                - mountPath: /home/amun/hwinfo
+                  name: hw-info-volume
+          initContainers:
+            - name: inspect-hwinfo
+              image: inspect-hwinfo
+              volumeMounts:
+                - mountPath: /home/amun/hwinfo
+                  name: hw-info-volume
+          volumes:
+            - name: hw-info-volume
+              emptyDir: {}
+      triggers:
+        - type: ImageChange
+          imageChangeParams:
+            automatic: true
+            containerNames: [inspect]
+            from:
+              kind: ImageStreamTag
+              name: '${AMUN_INSPECTION_ID}:latest'
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: CPU_FAMILY
+                    operator: In
+                    values:
+                      - ${CPU_FAMILY}
+                  - key: CPU_MODEL
+                    operator: In
+                    values:
+                      - ${CPU_MODEL}
+                  - key: PHYSICAL_CPUS
+                    operator: In
+                    values:
+                      - ${PHYSICAL_CPUS}
+                  - key: processor
+                    operator: In
+                    values:
+                      - ${PROCESSOR}
+
+parameters:
+  - name: AMUN_INSPECTION_ID
+    description: Id of inspection that is run.
+    displayName: Inspection id
+    required: true
+
+  - name: AMUN_CPU
+    description: CPU cores requested for build
+    displayName: Build CPU
+    default: 750m
+    required: true
+
+  - name: AMUN_MEMORY
+    description: Memory requested for build
+    displayName: Build Memory
+    default: 256Mi
+    required: true
+
+  - name: CPU_FAMILY
+    description: Family number of CPU.
+    displayName: CPU Family
+    required: true
```

Comment:
can we provide sane defaults? @sub-mod

Reply:
"cpufamily": "6", "cpumodel": "94" , "processor" : "Intel-Xeon-Processor-Skylake-IBRS" , cpu_physicalcpus : 32

Reply:
Adjusted.